.github/workflows/tests.yml

```diff
@@ -15,8 +15,14 @@ jobs:
     name: ${{ matrix.lisp }} on ${{ matrix.os }}
     strategy:
       matrix:
-        lisp: [sbcl-bin]
-        os: [ubuntu-latest]
+        # Use ccl-bin/1.12.1 instead of 'ccl' because of
+        # https://github.com/roswell/roswell/issues/534.
+        # TODO: Revert when Roswell is functional again.
+        lisp: [sbcl-bin, ccl-bin/1.12.1, ecl/21.2.1]
+        rosargs: [dynamic-space-size=3072]
+        os: [ubuntu-latest, macos-latest] # try windows-latest when we understand commands to install Roswell on it
+
+    # run the job on every combination of "lisp" and "os" above
```

Comment:
+1

Reply:
This PR changes thread management and one of the important features is to do it properly.  This could be highlighted by the tests not working properly on ECL and macOS.  Hence the need to restore the matrix to a wider state.

Reply:
@Ambrevar can you please put it in another, separate PR?

Reply:
Done in #37.